src/Models/Asset/AssetResolverInterface.php

```diff
@@ -0,0 +1,32 @@
+<?php
+declare(strict_types=1);
+
+/**
+ * Pimcore
+ *
+ * This source file is available under two different licenses:
+ * - GNU General Public License version 3 (GPLv3)
+ * - Pimcore Commercial License (PCL)
+ * Full copyright and license information is available in
+ * LICENSE.md which is distributed with this source code.
+ *
+ *  @copyright  Copyright (c) Pimcore GmbH (http://www.pimcore.org)
+ *  @license    http://www.pimcore.org/license     GPLv3 and PCL
+ */
+
+namespace Pimcore\Bundle\StaticResolverBundle\Models\Asset;
+
+use Pimcore\Model\Asset;
+use Pimcore\Model\Asset\Listing;
+
+/**
+ * @internal
+ */
+interface AssetResolverInterface
+{
+    public function getById(int|string $id, array $params = []): ?Asset;
```

Comment:
We should but a space between each line. 
Not sure if CS-Fixer is checking that?

Reply:
Done!